frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -764,7 +764,7 @@ function ReaderHighlight:onHoldRelease()
                         text = "Copy",
                         enabled = Device:hasClipboard(),
                         callback = function()
-                            Device.input.setClipboardText(self.selected_text.text)
+                            Device.input:setClipboardText(self.selected_text.text)
```

Comment:
The clipboard functions are actually functions and not methods, so this one (with parameter) should not work anymore as intended when switched from `.` to `:`. The others won't mind, but I guess they should still use `.`. (cf https://github.com/koreader/koreader/pull/4482#issuecomment-454492160 where I thought about making them methods, but didn't :)

Reply:
By "others" you mean Clipboard stuff, right? Basically, that only leaves hasClipboard`:` ;).

Reply:
Yep, (has|get|set)Clipboard(), that you restored as they were it seems, as they are no more in the diff.
Haven't looked at each of the Device:xxx if they are really a method, but mostly everything should be a method :)

Reply:
Yeah, I only rg'd for `evice\.`, so I shouldn't have touched anything else ;).